client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -125,11 +125,21 @@ class ShippingLabelViewWrapper extends Component {
 
 			// All items are packaged, just show track button
 			return (
-				<Button
-					onClick={ this.handleTrackPackagesButtonClick }
-				>
-					{ translate( 'Track Packages' ) }
-				</Button>
+				<span> 
```

Comment:
@budzanowski given this logic now, I think we can get rid of L105-L124. Do you agree?

Reply:
Not really. Although the functionality is virtually the same, the labels are a tiny bit different but the styling is totally different. This is done to emphasize functionality importance in different scenarios. We could add logic around scenarios to present different styling but that would be equally awkward imo. I don't want to sacrifice code readability for a couple of lines of code. I don't have a very strong opinion on this, it just feels right this way for me.